simple_rl/mdp/MDPClass.py

```diff
@@ -40,6 +40,9 @@ def get_transition_func(self):
     def get_num_state_feats(self):
         return self.init_state.get_num_feats()
 
+    def get_slip_prob(self):
```

Comment:
Same point from before: it's best to keep things like this out of the MDP since it's pretty MDP specific.